src/main/scala/extractionStreams/WikiExtractionStream.scala

```diff
@@ -0,0 +1,14 @@
+package extractionStreams
+
+import akka.stream.Materializer
+import akka.stream.scaladsl.RunnableGraph
+
+/**
+ * Created by nico on 13/07/15.
+ */
+abstract class WikiExtractionStream[T] {
+  val stream: RunnableGraph[T]
+  def run()(implicit materializer: Materializer): Unit = {
```

Comment:
this is a good introduction: http://engineering.intenthq.com/2015/06/wikidata-akka-streams/